tests/e2e-pw/specs/shopper/shopper-subscriptions-purchase-free-trial.spec.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import test, { expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { shouldRunSubscriptionsTests } from '../../utils/constants';
+import { describeif, getMerchant, getShopper } from '../../utils/helpers';
+import RestAPI from '../../utils/rest-api';
+import { config } from '../../config/default';
+import {
+	confirmCardAuthentication,
+	emptyCart,
+	fillCardDetails,
+	setupCheckout,
+} from '../../utils/shopper';
+import {
+	goToCart,
+	goToProductPageBySlug,
+} from '../../utils/shopper-navigation';
+import { goToOrder, goToSubscriptions } from '../../utils/merchant-navigation';
+
+const nowLocal = new Date();
+const nowUTC = new Date(
+	nowLocal.getUTCFullYear(),
+	nowLocal.getUTCMonth(),
+	nowLocal.getUTCDate()
+);
+const formatter = new Intl.DateTimeFormat( 'en-US', {
+	dateStyle: 'long',
+} );
+const renewalDate = nowUTC.setDate( nowUTC.getDate() + 14 );
+const renewalDateFormatted = formatter.format( renewalDate );
+const productName = 'Subscription free trial product';
+const productSlug = 'subscription-free-trial-product';
+const customerBilling = config.addresses.customer.billing;
+const customerBillingConfig =
+	config.addresses[ 'subscriptions-customer' ].billing;
+let orderId, subscriptionId;
```

Comment:
Nitpick: let's set these variables' types.

```
let orderId: string, subscriptionId: string;
```

Reply:
Thanks for the reminder, I've set them in the last iteration.